templates/store/layout.html.twig

```diff
@@ -51,6 +51,11 @@
             {{ 'stores.deliveries'|trans }}
           </a>
         </li>
+        <li role="presentation">
+          <a href="{{ path('admin_store_deliveries_bookmarks', { id: store.id }) }}">
+            {{ 'stores.deliveries.bookmarks'|trans }}
```

Comment:
would that be possible to add a small help text?

Reply:
Added an empty state message. Will check on content with Vincent next week